install_config/router/f5_router.adoc

```diff
@@ -243,27 +241,27 @@ BIG-IP® documentation] for more information.
 . xref:f5-configuring-the-virtual-servers[Configure a virtual server] for the
 custom partition.
 
-. Deploy the F5 router using the `--external-host-partition-path` flag to specify a partition path:
+. Deploy the F5 router plug-in using the `--external-host-partition-path` flag to specify a partition path:
 +
 ----
 $ oc adm router --external-host-partition-path=/OpenShift/zone1 ...
 ----
 
 
 [[setting-up-f5-native-integration-with-openshift]]
-== Setting Up F5 Native Integration
+== Setting Up F5 Router Plug-in
 
 [NOTE]
 ====
 This section reviews how to set up F5 native integration with {product-title}.
-The concepts of F5 appliance and {product-title} connection and data flow of F5
-native integration are discussed in the
-xref:../../architecture/networking/assembly_available_router_plugins.adoc#architecture-f5-native-integration[F5 Native Integration] section of the Routes topic.
+The concepts of the F5 appliance and {product-title} connection and data flow of the F5
+router plug-in are discussed in the
+xref:../../architecture/networking/assembly_available_router_plugins.adoc#architecture-f5-native-integration[F5 Router Plug-in] section of the Routes topic.
```

Comment:
s/F5 Router Plug-in/F5 Native Integration

Please review https://docs.openshift.com/container-platform/3.10/architecture/networking/assembly_available_router_plugins.html#architecture-f5-native-integration for other changes to suggest to that topic. It has more references to "F5 Native Integration," and it looks like you want to remove that phrase.